documentation/Architecture Rules.adoc

```diff
@@ -0,0 +1,6 @@
+== Architecture Rules
+
+The following image illustrates the devonfw architecture rules. The arrows show the allowed dependencies in green, discouraged dependencies in orange and forbidden dependencies in red.
+image:DevonArchitectureRules.png["Devon Architecture Rules",link=DevonArchitectureRules.png]
```

Comment:
Careful with pictures, you should relocate them into the documentation folder so they can be found from the asciidoc. I would recommend creating a seperate folder for images inside the documentation folder. The value of 'link' would then be "images/[name_of_picture].png"